include/RAJA/policy/cuda/sort.hpp

```diff
@@ -46,39 +46,39 @@ namespace sort
 */
 template <size_t BLOCK_SIZE, bool Async, typename Iter, typename Compare>
 concepts::enable_if<concepts::negate<concepts::all_of<
-                      type_traits::is_arithmetic<detail::IterVal<Iter>>,
+                      type_traits::is_arithmetic<RAJA::detail::IterVal<Iter>>,
                       std::is_pointer<Iter>,
                       concepts::any_of<
-                        camp::is_same<Compare, operators::less<detail::IterVal<Iter>>>,
-                        camp::is_same<Compare, operators::greater<detail::IterVal<Iter>>>>>>>
+                        camp::is_same<Compare, operators::less<RAJA::detail::IterVal<Iter>>>,
+                        camp::is_same<Compare, operators::greater<RAJA::detail::IterVal<Iter>>>>>>>
 stable(const ::RAJA::cuda_exec<BLOCK_SIZE, Async>&,
      Iter,
      Iter,
      Compare)
 {
   static_assert(concepts::all_of<
```

Comment:
Having just one `static_assert` makes digesting the exact error harder. Using multiple static asserts will inform the user exactly of the error they have:

```cpp
static_assert (std::is_pointer<Iter>::value, 
  "stable_sort<cuda_exec> is only implemented for pointers");
using iterval = RAJA::detail::IterVal<Iter>;
static_assert (type_traits::is_arithmetic<iterval>::value,
  "stable_sort<cuda_exec> is only implemented for arithmetic types");
static_assert (concepts::any_of<
  camp::is_same<Compare, operators::less<iterval>>,
  camp::is_same<Compare, operators::less<iterval>>>::value,
  "stable_sort<cuda_exec> is only implemented for RAJA::operators::less or RAJA::operators::greater");
```

Reply:
fixed